app/org/sagebionetworks/bridge/services/email/MimeTypeEmailBuilder.java

```diff
@@ -37,8 +38,13 @@ MimeTypeEmailBuilder withSender(String senderAddress) {
      * @param recipientAddress
      * @return
      */
-    MimeTypeEmailBuilder withRecipient(String recipientAddress) {
-        this.recipientAddresses.add(recipientAddress);
+    /**
+     * A recipient for this email (may call this more than once and values will be accumulated).
+     * @param recipientAddress
+     * @return
+     */
+    MimeTypeEmailBuilder withRecipients(Collection<String> recipients) {
+        this.recipientAddresses.addAll(recipients);
         return this;
     }
```

Comment:
Would it be cleaner to have both withRecipient() and withRecipients()? That way, you don't have to create singleton sets, and you don't have to do weird set manipulation.